src/DataContext.js

```diff
@@ -22,6 +22,13 @@ function onlyShowEventsWithImages(events) {
   return R.filter(event => event.image !== null, events);
 }
 
+function useHttpsInImages(events) {
+  return events.map(event => {
+    event.image = event.image.replace(/^http:\/\//i, 'https://');
+    return event
+  })
+}
```

Comment:
I think this could also work:

```javascript
const useHttpsInImages = (events) => 
  events.map(event => ({ 
    ...event,
    image: event.image.replace(/^http:\/\//i, 'https://')
  }));
```

but perfectly valid as given :)

Reply:
I used your approach, looks nicer. However, another return statement before `events.map(...)` was needed to get it working

Reply:
Ah yeah, maybe the `event` should have parentheses around to work. The ES6 syntax is sometimes somewhat weird ;D